website/public/install/common-manual-tabs.ejs.html

```diff
@@ -0,0 +1,114 @@
+<%#
+<!-- Copyright (C) 2020  Matthew "strager" Glazar -->
+<!-- See end of file for extended copyright information. -->
+%>
+
+<style>
+  #tab-bar {
+    display: none;
+  }
+
+  #tab-bar button {
+    display: table-header-group;
+    padding: 6px 12px;
+    border: none;
+    border-bottom: 4px solid #1b1b1b;
+    background-color: #bdbdbd;
+    transition: 0.3s;
+    font-size: medium;
+
+    -webkit-user-select: text;
+    -moz-user-select: text;
+    -ms-user-select: text;
+    user-select: text;
+  }
+
+  #tab-bar button:hover {
+    background-color: #858585;
+  }
+
+  @media (prefers-reduced-motion) {
+    #tab-bar button {
+      transition: 0ms;
+    }
+  }
+
+  #package-list-header {
+    font-size: medium;
+    border-bottom: #b9b9b9 2px solid;
+    padding-bottom: 4px;
+    margin-bottom: 16px;
+    color: #2b2b2b;
+  }
+
+  .package-managers {
+    border-left: 1px #858585 solid;
+    padding-left: 8px;
+  }
+
+  @media (prefers-color-scheme: dark) {
+    #package-list-header {
+      color: #b9b9b9;
+    }
+
+    #tab-bar button {
+      color: #1b1b1b;
+      background-color: #727272;
+    }
+  }
+</style>
+
+<script>
+  var borderColorHighlighted = "red";
+  var borderColorDefault = "black";
+
+  document.addEventListener("DOMContentLoaded", function () {
+    changePackageManager(null, "none");
+    document.getElementById("tab-bar").style.display = "table-header-group";
+  });
+
+  function changePackageManager(event, selectedManager) {
+    let tabContent = document.getElementsByClassName("package-managers");
+    for (i = 0; i < tabContent.length; i++) {
```

Comment:
`i` is a global variable. Make it a local variable instead:
```suggestion
    for (let i = 0; i < tabContent.length; i++) {
```